### ISSUE TITLE
Exception swallowing during page.insertImage(pixmap)

### ISSUE THREAD

**Issue by joerobmunoz** [pymupdf/PyMuPDF#302]
Hey,

Thanks again for the package, it's been working great. Just wanted to report an issue I saw while writing a picture.

When trying to use the ```page.insertImage``` function that uses the pixmap object, I've run into an issue that crashes the interpreter and doesn't seem to spit out any diagnostic information.

```python
import fitz
import numpy as np

doc = fitz.open()
page = doc.newPage()
height, width = 100, 100
samples = np.random.randint(0, 255, (height, width, 3), dtype=np.uint8).tostring() # get plain pixel data from numpy array
pix = fitz.Pixmap(fitz.csRGB, width, height, samples)

rect = fitz.Rect(0, 0, 100, 100)
page = doc.newPage()
page.insertImage(rect, pixmap = pix)
```

> fitz.version
('1.14.13', '1.14.0', '20190407064320')

I've bypassed this method by using ```page.insertImage(rect, file="that_same_image.png")```, having the pixmap write to an intermediate file, and it works perfectly.

I'd be interested to learn any tips anyone has in debugging things like this in the future, since I'm a bit rusty on the C -> Python pieces.

**Comment by JorjMcKie** [comment]
This issue has been solved in the next version 1.14.14.
Which I unfortunately did not upload to PyPI. But you can download the Windows version from the releases folder of this repo and Linux / OSX  versions (even 1.14.15) from [here](https://github.com/pymupdf/PyMuPDF-wheels).
The reason for this irregular situation is that I am currently working on v1.15.0 - MuPDF has recently published the corresponding base lib version.
Just tested your example with v1.14.14 and 1.14.15 - both are working.

You realize, that you are creating 2 pages in your example? And the picture is on page 2 in case you wonder where it landed :-)

**Comment by joerobmunoz** [comment]
Thanks for the quick reply! I'll look into grabbing that new package (also 1.15). 

The page stuff was clipped from a unit test that I'm writing, so it was intentional. Thanks for reading that closely, ha!
